lib/src/commands/Deprecations.ts

```diff
@@ -1,7 +1,65 @@
 import once from 'lodash/once';
+import get from 'lodash/get';
+import each from 'lodash/each';
 import { Platform } from 'react-native';
 
 export class Deprecations {
+  private deprecatedMethods: Array<{ key: string; showWarning: any }> = [
```

Comment:
Perhaps rename to `deprecatedOptions`?

```suggestion
  private deprecatedOptions: Array<{ key: string; showWarning: any }> = [
```